prediction_market_agent/agents/microchain_agent/nft_treasury_game/scripts/reset_balance_anvil.py

```diff
@@ -26,5 +34,39 @@ def main(
     )
 
 
-if __name__ == "__main__":
-    typer.run(main)
+def get_token_owner(token_id: int, web3: Web3) -> ChecksumAddress:
+    nft_contract = SimpleTreasuryContract().nft_contract(web3=web3)
+    return Web3.to_checksum_address(nft_contract.owner_of(token_id=token_id, web3=web3))
```

Comment:
`owner_of` already return checsumaddress 

```suggestion
    return nft_contract.owner_of(token_id=token_id, web3=web3)
```